AuthenticationExample/AuthenticationExample/AuthenticationApp.swift

```diff
@@ -18,19 +18,9 @@ import ArcGIS
 
 @main
 struct AuthenticationApp: App {
-    @ObservedObject var authenticator: Authenticator
-    @State var isSettingUp = true
+    @StateObject private var authenticator = Authenticator()
     
-    init() {
-        // Create an authenticator.
-        authenticator = Authenticator(
-            // If you want to use OAuth, uncomment this code:
-            //oAuthUserConfigurations: [.arcgisDotCom]
```

Comment:
When this is uncommented `.arcgisDotCom` is undefined, leaving the developer with an example that won't compile. Instead I've opted to mention the alternate initializer in the component's tutorial.